graylog2-web-interface/src/views/components/visualizations/XYPlot.jsx

```diff
@@ -50,14 +47,14 @@ const XYPlot = ({
   config,
   chartData,
   currentQuery,
-  timezone,
   effectiveTimerange,
   getChartColor,
   setChartColor,
   height,
   plotLayout = {},
   onZoom = OnZoom,
 }: Props) => {
+  const { timezone } = useContext(CurrentUserContext) || { timezone: 'UTC' };
```

Comment:
`graylog2-web-interface/config.js` shows that the default timezone is 'Europe/Berlin'. Not sure if that's relevant. Also, should we hoist this default up into the context's default?

Reply:
I've implemented the `AppConfig` `rootTimeZone`  as a default fallback. I prefer to define the fall back inside the component, because we check e.g. in the `App.jsx` if the `currentUser` object is empty.